Vagrantfile

```diff
@@ -64,10 +98,15 @@ Vagrant.configure('2') do |config|
 
   if ENV['BOLT_TEST_USE_VAGRANT']
     config.vm.define :linux do |linux|
-      linux.vm.box = 'bento/centos-6.7'
-      linux.vm.network :forwarded_port, guest: 22, host: 20022, host_ip: '127.0.0.1', id: 'ssh'
+      linux.vm.box = 'generic/ubuntu2004'
+      linux.vm.network :forwarded_port, guest: 22, host: 20023, host_ip: '127.0.0.1', id: 'ssh'
```

Comment:
This is the same port used by one of our docker containers. I think you'd want 20026?